build.rs

```diff
@@ -0,0 +1,25 @@
+use std::process::Command;
+
+fn main() {
```

Comment:
This build script should probably only run if the dockerfile changes: https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed

Reply:
Cool, I almost wrote this myself 😅